src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,31 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao.impl");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book1 = new Book();
+        book1.setTitle("Mindset");
+        book1.setPrice(BigDecimal.valueOf(190));
+
+        System.out.println(bookDao.findById(2L).get());
+
+        System.out.println(bookDao.create(book1));
+
+        Book book2 = new Book();
```

Comment:
```suggestion
        Book secondBook = new Book();
```

Try to avoid numbers in naming